lib/src/androidTest/java/com/sensirion/libble/devices/BleDeviceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.sensirion.libble.devices;
+
+import android.test.AndroidTestCase;
+import android.test.suitebuilder.annotation.SmallTest;
+
+import com.sensirion.libble.services.BleService;
+import com.sensirion.libble.services.MockBleService;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static com.sensirion.libble.services.BleServiceSynchronizationPriority.HIGH_PRIORITY;
```

Comment:
(applies to every static import of this commit) import the enum, not each enum value individually and use it with the enum's name to give context: BleServiceSynchronizationPriority.HIGH_PRIORITY